### PR TITLE
Updated meta titles for SEO

### PR DESCRIPTION
**Pull request by DrParadox05**
## Description
Updated the meta-titles of the requested pages. Fixed: #13796

### CODE REVIEW

**Comment by konopkja** [comment]
@minimalsm pls review

File: public/content/dao/index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Decentralized autonomous organizations (DAOs)
+title: What is a DAO? | Decentralized Autonomous Organization
```
Comment (minimalsm) [inline public/content/dao/index.md:2]:
```suggestion
title: What is a DAO? | Decentralized Autonomous Organizations
```

File: public/content/developers/docs/gas/index.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Gas and fees
+title: "Ethereum gas and fees: technical overview"
```
Comment (minimalsm) [inline public/content/developers/docs/gas/index.md:2]:
```suggestion
title: Ethereum gas and fees: technical overview
```

File: public/content/smart-contracts/index.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Smart contracts
+title: "Smart contracts: What are they and benefits"
```
Comment (minimalsm) [inline public/content/smart-contracts/index.md:2]:
```suggestion
title: Smart contracts: What are they and benefits
```

**Review by minimalsm — APPROVED** [review]
LGTM, thanks @DrParadox05!

@konopkja there is a chance by changing the JSON files, we could be using these strings in other places (like the footer, see image below), but I'll let you be the judge of whether that is important in this PR or whether we need to deal with that later.

Screenshot 2024-09-16 at 17 15 20

**Review by konopkja — APPROVED** [review]

**Pull request merged**